### DISCUSSION TITLE
Format postgresql timestampz in a table

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6423] [Help] [answered]
My timestampz "Date début" is 2022-06-21 00:00:00.000 +0200

If I add a tag in my table with the syntax {{ date tbl_ruptures.RUPTURES.Date début "YYYY-MM-DD" }}  I get the correct answer : "2022-06-21" 👍 

Now in my table if I put {{ date Value "YYYY-MM-DD" }} I get "June 21 2022,00:00"  👎

**Comment by melohagan** [comment, marked as answer]
Thanks @NicolasFabre81 

I see what you mean:
![Screenshot 2022-06-22 at 16 48 47](https://user-images.githubusercontent.com/101575380/175075665-4505b077-427d-4c77-8544-10e153f90ce2.png)

----

What you can do to get around this is to add a <a href="https://docs.budibase.com/docs/formula">formula</a> column to your table.

![Screenshot 2022-06-22 at 16 51 15](https://user-images.githubusercontent.com/101575380/175076142-90d2acf6-5e4f-4a83-94ed-92adb645a1d9.png)

![Screenshot 2022-06-22 at 16 51 54](https://user-images.githubusercontent.com/101575380/175076278-dff389d5-d7bc-4046-a90b-8cd4a07e25f8.png)

**Reply by NicolasFabre81** [reply]
I'm sorry but I dont' see formula in my column list ?? I only see the list of my data provider fields..
![image](https://user-images.githubusercontent.com/28592069/175081576-fd3c21bb-ed78-471d-859b-643f8e1a04be.png)

**Reply by melohagan** [reply]
Formula columns are in the **Data** tab (not the designer)

**Reply by NicolasFabre81** [reply]
GREAT it works ! But I'm afraid... As I use postgreSQL, where is stored this new column ?
Don't I risk to loose it, for example if I re-fetch my tables ?

**Reply by melohagan** [reply]
The column is stored in your Budibase app. I tested this myself, and the re-fetch does not appear to remove the formula column.

**Reply by NicolasFabre81** [reply]
I also tested : just fetching is OK, but if you change your data source, for example to switch from sandbox to production, then the formula column disappear ! In my case the best will be to change the data type from timestamp to date.
